ahds/header.py

```diff
@@ -39,65 +114,165 @@
     data_index: 1
     data_length: None
 
-Data pointers are identified by the name ``data_pointer_<n>``.
+Data pointers are identified by the name ``data_pointer_<n>`` where <n> is the number indicated by the
+block attibute in the above examples.
+
+The above approach can also be used to load Amira HyperSurface files.
+
+    >>> ah = AmiraHeader.from_file('<somfile>.surf')
+
+The restulting structure and behavior is the same as for AmiraMesh files.
+
+    >>> print(ah)
+    AMIRA HEADER (<somfile>.surf)
```

Comment:
Very good idea to show nested blocks but some standardisation would make it easy for the user to understand.

Reply:
I'm with you, any suggestions?